app/controllers/api/auth_controller.rb

```diff
@@ -1,58 +1,64 @@
 class Api::AuthController < Api::ApiController
-
   skip_before_action :authenticate_user, except: [:change_pw, :update]
 
   before_action do
     # current_user can still be nil by here.
     user = User.find_by_email(params[:email])
-    if user && user.locked_until && user.locked_until.future?
-      render :json => {error: {message: "Too many successive login requests. Please try your request again later."}}, :status => 423
+    if user&.locked_until && user.locked_until.future?
+      render json: {
+        error: {
+          message: 'Too many successive login requests. '\
+            'Please try your request again later.',
+        },
+      }, status: 423
     end
     @user_manager = user_manager
   end
 
   def mfa_for_email(email)
     user = User.find_by_email(email)
-    return if user == nil
-    mfa = user.items.where("content_type" => "SF|MFA", "deleted" => false).first
-    return mfa
+    return if user.nil?
+
+    user.items.where(content_type: 'SF|MFA', deleted: false).first
   end
 
-  def verify_mfa(email)
+  def verify_mfa(_email)
```

Comment:
If `email` is unused we should just remove this argument and also update function invocations to not take a parameter.